.github/workflows/main_and_pr_workflow.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        java-version: [ 11, 17, 21 ]
+        java-version: [ 11, 17, 21, 23 ]
```

Comment:
23 is online now? And we just skip 22 since it is deprecated? 😅

Reply:
Yes! But this is GA, I will work on our runners :)

Reply:
Yeah, 22 is out of the game